src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.targets

```diff
@@ -9,7 +9,9 @@
     <AndroidApplication Condition=" '$(AndroidApplication)' == '' And '$(OutputType)' == 'Exe' ">true</AndroidApplication>
     <AndroidApplication Condition=" '$(AndroidApplication)' == '' ">false</AndroidApplication>
     <SelfContained Condition=" '$(SelfContained)' == '' And '$(AndroidApplication)' == 'true' ">true</SelfContained>
-    <RuntimeIdentifier Condition=" '$(RuntimeIdentifier)' == '' And '$(AndroidApplication)' == 'true' ">android.21-x86</RuntimeIdentifier>
+    <!-- Prefer $(RuntimeIdentifiers) plural -->
+    <RuntimeIdentifiers Condition=" '$(RuntimeIdentifier)' == '' And '$(RuntimeIdentifiers)' == '' And '$(AndroidApplication)' == 'true' ">android.21-arm;android.21-arm64</RuntimeIdentifiers>
```

Comment:
The default `$(RuntimeIdentifiers)` are arm32 & arm64?  Why?  Especially considering that the *previous* default was x86, and the "normal" Xamarin.Android default -- via xamarin/monodroid, for commercial users -- is "all the ABIs!" `arm64-v8a;armeabi-v7a;x86;x86_64`.

Given that (1) we don't know what the "commercial integration" looks like, and (2) it would be Really Really Handy to be able to run the `.apk`s on x86(_64?) emulators, this default seems "weird" to me.

Reply:
This is the same default we have here: https://github.com/xamarin/xamarin-android/blob/5a5580cd3cce0559da5e9d097b03d356a6630c05/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L234

I can put them all, if we think that makes more sense.

Reply:
This now defaults to `android.21-arm64;android.21-x86`.